public/app/plugins/panel/canvas/editor/element/utils.ts

```diff
@@ -20,15 +23,12 @@ export const callApi = (api: APIEditorConfig, isTest = false) => {
       .fetch(request)
       .subscribe({
         error: (error) => {
-          if (isTest) {
-            appEvents.emit(AppEvents.alertError, ['Error has occurred: ', JSON.stringify(error)]);
-            console.error(error);
-          }
+          appEvents.emit(AppEvents.alertError, ['Error has occurred: ', JSON.stringify(error)]);
+          updateLoadingState && updateLoadingState(false);
         },
         complete: () => {
-          if (isTest) {
-            appEvents.emit(AppEvents.alertSuccess, ['Test successful']);
-          }
+          appEvents.emit(AppEvents.alertSuccess, ['Test successful']);
```

Comment:
I think we might want to update these messages as now it not just the "test" scenario 🤔 
```suggestion
          appEvents.emit(AppEvents.alertSuccess, ['Canvas button API call was successful']);
```

Reply:
maybe this doesn't have to be as explicit 🤷‍♂️

Reply:
oops, missed this. Maybe `API call was successful`?

Reply:
sure sounds good 👍